src/clang-c-frontend/clang_c_convert.cpp

```diff
@@ -547,20 +547,26 @@ bool clang_c_convertert::get_var(const clang::VarDecl &vd, exprt &new_expr)
      * initializer can't have side-effects (it's a constant expression). */
     code_blockt *orig = current_block;
     current_block = nullptr;
+    const clang::Stmt *stmt = vd.getInit();
```

Comment:
Does it generalize to direct list initialization? Would it be possible for you to try the followings? 
```
dim3 s1 = {1, 2, 3}; // copy list initialization
dim3 s1 {1, 2, 3}; // direct list initialization
```

Reply:
Hi @kunjsong01 ,

Our compiler doesn't seem to support some C++ 11 features, and I get the following error message when I try to confirm the AST:
```
ESBMC version 7.3.0 64-bit x86_64 linux
Target: 64-bit little-endian x86_64-unknown-linux with esbmclibc
Parsing main2.cpp
main2.cpp:8:9: error: expected ';' after top level declarator
uint3 s1 {1, 2, 3};
        ^
        ;
ERROR: PARSING ERROR
```
I tested it in the latest version.